doc/semitrans.xml

```diff
@@ -349,3 +349,29 @@ gap> FixedPointsOfTransformationSemigroup(S);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="WreathProduct">
+  <ManSection>
+    <Oper Name = "WreathProduct" Arg = "S, G"/>
+    <Returns>The wreath product of <A>S</A> and <A>G</A>.</Returns>
```

Comment:
The `Returns` statement should just be the type of object returned: `A transformation semigroup.`